css/index.css

```diff
@@ -270,81 +47,6 @@ section {
   vertical-align: middle !important;
 }
 
-.no-underline-hover:hover {
-  text-decoration: none;
-}
-
-.splash-image {
-  width: 30%;
-  height: auto;
-  margin-left: auto;
-  margin-right: auto;
-  display: block;
-}
-
-.section-heading {
-}
-
-.splash-text-large {
-  text-align: center;
-  font-size: 64px;
-}
-
-.splash-text-small {
-  text-align: center;
-  font-size: 24px;
-}
-
-.section-paragraph {
-
-}
-
-.example-paragraph {
-  border-top: 1px solid #e8e8e8;
-  border-bottom: 1px solid #e8e8e8;
-  padding: 10px;
-}
-
-.section-paragraph-image {
-  width: auto;
-  height: 82px;
-  float: left;
-  margin-right: 10px;
-  clear:both;
-}
-
-.right-of-image {
-  margin-left: 0px;
-  overflow:auto;
-  display:block;
-}
-
-@media only screen and (max-width: 1024px) {
-  .splash-text-large {
-    font-size: 48px;
-  }
-
-  .splash-text-small {
-    font-size: 18px;
-  }
-}
-
-@media only screen and (max-width: 720px) {
-  .splash-text-large {
-    font-size: 32px;
-  }
-
-  .splash-text-small {
-    font-size: 14px;
-  }
-}
-
-@media only screen and (max-width: 360px) {
-  .splash-text-large {
-    font-size: 24px;
-  }
-
-  .splash-text-small {
-    font-size: 12px;
-  }
-}
+svg {
+    padding-bottom: 50px;
+}
```

Comment:
Nit: File should end with a newline character.

Reply:
Fixed in 71136f8